site/src/pages/release-notes/1.29.0.mdx

```diff
@@ -0,0 +1,257 @@
+---
+date: 2024-06-05
+---
+
+## 🌟 New features
+
+- Armeria [HTTP/JSON to GRPC transcoding](https://cloud.google.com/endpoints/docs/grpc/transcoding) service now
+  supports [Custom methods](https://cloud.google.com/apis/design/custom_methods). #5613
+  ```protobuf
+  rpc Watch(WatchRequest) returns (WatchResponse) {
+    option (google.api.http) = {
+      post: "/v1:watch" // 👈👈👈
+      body: "*"
+    };
+  }
+  ```
+- You can now specify when to remove multipart temporary files using <type://MultipartRemovalStrategy>. #5652 #5653
```

Comment:
Optional) It may be worth noting that the default has changed to `ON_RESPONSE_COMPLETION` as well.